types/jsonwebtoken/index.d.ts

```diff
@@ -159,12 +160,12 @@ export function sign(
  * callback - Callback to get the encoded token on
  */
 export function sign(
-    payload: string | Buffer | object,
+    payload: string | Buffer | { [key: string]: any },
     secretOrPrivateKey: Secret,
     callback: SignCallback,
 ): void;
 export function sign(
-    payload: string | Buffer | object,
+    payload: string | Buffer | { [key: string]: any },
```

Comment:
```suggestion
    payload: string | Buffer | { [key: string]: unknown },
```

Same reason as above.